dist/angular-cron-jobs.js

```diff
@@ -324,52 +328,52 @@ angular.module('angular-cron-jobs').factory('cronService', function() {
 
         if (cron[0] !== '*') {
             //preparing to handle multiple minutes
-            if (cron[0].indexOf(',') >= 0) {
+            if (allowMultiple) {
                 var tempArray = cron[0].split(',');
                 for (var i = 0; i < tempArray.length; i++) { tempArray[i] = +tempArray[i]; }
-                frequency.minuteValue = tempArray;
+                frequency.minuteValues = tempArray;
             } else {
-                frequency.minuteValue = [parseInt(cron[0])];
+                frequency.minuteValues = [parseInt(cron[0])];
             }
         }
         if (cron[1] !== '*') {
             //preparing to handle multiple hours
-            if (cron[1].indexOf(',') >= 0) {
+            if (allowMultiple) {
                 var tempArray = cron[1].split(',');
                 for (var i = 0; i < tempArray.length; i++) { tempArray[i] = +tempArray[i]; }
-                frequency.hourValue = tempArray;
+                frequency.hourValues = tempArray;
             } else {
-                frequency.hourValue = [parseInt(cron[1])];
+                frequency.hourValues = parseInt(cron[1]);
             }
         }
         if (cron[2] !== '*') {
             //preparing to handle multiple days of the month
-            if (cron[2].indexOf(',') >= 0) {
+            if (allowMultiple) {
                 var tempArray = cron[2].split(',');
                 for (var i = 0; i < tempArray.length; i++) { tempArray[i] = +tempArray[i]; }
-                frequency.dayOfMonthValue = tempArray;
+                frequency.dayOfMonthValues = tempArray;
             } else {
-                frequency.dayOfMonthValue = [parseInt(cron[2])];
+                frequency.dayOfMonthValues = [parseInt(cron[2])];
             }
         }
         if (cron[3] !== '*') {
             //preparing to handle multiple months
-            if (cron[3].indexOf(',') >= 0) {
+            if (allowMultiple) {
                 var tempArray = cron[3].split(',');
                 for (var i = 0; i < tempArray.length; i++) { tempArray[i] = +tempArray[i]; }
-                frequency.monthValue = tempArray;
+                frequency.monthValues = tempArray;
             } else {
-                frequency.monthValue = [parseInt(cron[3])];
+                frequency.monthValues = [parseInt(cron[3])];
             }
         }
         if (cron[4] !== '*') {
             //preparing to handle multiple days of the week
-            if (cron[4].indexOf(',') >= 0) {
+            if (allowMultiple) {
                 var tempArray = cron[4].split(',');
                 for (var i = 0; i < tempArray.length; i++) { tempArray[i] = +tempArray[i]; }
-                frequency.dayValue = tempArray;
+                frequency.dayValues = tempArray;
             } else {
-                frequency.dayValue = [parseInt(cron[4])];
+                frequency.dayValues = [parseInt(cron[4])];
```

Comment:
Yeah (for all non-array comments) I have some updates I will push as soon as i get the tests working.